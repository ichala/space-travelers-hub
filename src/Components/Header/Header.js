import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../../images/logo.png';

function Header() {
  return (
    <nav>
      <div className="logo">
        <img className="logo-header" src={logo} alt="logo" />
        <h4>Space Travelers Hub</h4>
      </div>
      <div className="links">
        <NavLink className="link" to="/"> Missions</NavLink>
        <NavLink className="link" to="/Dragons"> Dragons</NavLink>
        <NavLink className="link" to="/Rockets"> Rockets</NavLink>
        {' '}
        |
        <NavLink className="link" to="/Profile"> Profile</NavLink>
      </div>
    </nav>
  );
}

export default Header;
