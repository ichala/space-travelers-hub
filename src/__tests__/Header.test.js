import { HashRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Header from '../Components/Header/Header';

it('Header Components Renders correctly', () => {
  const tree = renderer.create(
    <Router><Header /></Router>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
