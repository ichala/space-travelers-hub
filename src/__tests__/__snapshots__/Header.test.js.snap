// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Components Renders correctly 1`] = `
<nav>
  <div
    className="logo"
  >
    <img
      alt="logo"
      className="logo-header"
      src="logo.png"
    />
    <h4>
      Space Travelers Hub
    </h4>
  </div>
  <div
    className="links"
  >
    <a
      aria-current="page"
      className="link active"
      href="#/"
      onClick={[Function]}
    >
       Missions
    </a>
    <a
      className="link"
      href="#/Dragons"
      onClick={[Function]}
    >
       Dragons
    </a>
    <a
      className="link"
      href="#/Rockets"
      onClick={[Function]}
    >
       Rockets
    </a>
     
    |
    <a
      className="link"
      href="#/Profile"
      onClick={[Function]}
    >
       Profile
    </a>
  </div>
</nav>
`;
